src/app.md

```diff
@@ -134,7 +134,31 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![syslog-badge]][syslog] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+Messages are logged to [UNIX syslog]. Logger backend is initialized
+with [`syslog::init`].
+[`syslog::Facility`] indicates type of program submitting log.
+[`log::LogLevelFilter`] denotes alowed log verbosity
+and `Option<&str>` holds optional aplication name.
+
+```rust,no_run
```

Comment:
Thanks for your time :) 
it is marked as `no_run` in order not to clobber the syslog. I guess that most people would prefer not to have noise in their syslog at every testrun.